### PR TITLE
Removing license restrictions

### PR DESCRIPTION
**Pull request by javaguirre**
#### Summary

MS Teams integration can be used with any Mattermost license now, removing License restriction code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,7 @@ Visit the [MS Teams Meetings documentation](https://mattermost.gitbook.io/micros
 
 ## License
 
-This repository is licensed under the [Mattermost Source Available License](LICENSE), and requires a valid Mattermost E20, Professional, or Enterprise license. See our [documentation](https://docs.mattermost.com/about/frequently-asked-questions.html#mattermost-source-available-license) to learn more. If you are contributing to the project, please enable [ServiceSettings.EnableDeveloper](https://docs.mattermost.com/configure/configuration-settings.html#enable-developer-mode) in your server's config. The plugin will not start and show an error message in your server logs, if you are the missing the Enterprise License.
-
+This repository is licensed under the [Mattermost Source Available License](LICENSE).
```
[the remark this answers is not in the document]
Comment (javaguirre) [reply README.md]: Done!

**Comment by catalintomai** [comment]
@javaguirre , if there isn't one open, please open a Jira ticket to capture the same and link here. Might be needed for reporting in Product, other places. Thanks! cc: @asatkinson

**Review by larkox — APPROVED** [review]

**Comment by DHaussermann** [comment]
@hanzei and @javaguirre I'll take a look. But I believe we also need a corresponding marketplace PR to include the plugin in the marketplace result set.

**Comment by javaguirre** [comment]
Didn't know that, I'll check how that works, thank you! 👍

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Tested no license applied
- Tested Starter, Professional and Enterprise 
- Tested On-Prem and Cloud
- No issues Enabling the plugin in any cases.

LGTM!

**Review by asatkinson — APPROVED** [review]
Ticket Link = https://mattermost.atlassian.net/browse/MM-48541?atlOrigin=eyJpIjoiZjgwZTI2OWU4OGNiNDY4ZjlmZWUxNWZkNTQzYjI1ZDQiLCJwIjoiaiJ9

**Pull request merged**